libs/features/mas/src/merch-card.js

```diff
@@ -113,6 +114,7 @@ export class MerchCard extends LitElement {
         this.filters = {};
         this.types = '';
         this.selected = false;
+        this.spectrum = 'css';
```

Comment:
We could document it here:
https://mwpw-163479--milo--adobecom.aem.live/libs/features/mas/docs/merch-card.html#attributes

Reply:
updated the doc

Reply:
thx!